packages/babel-plugin-transform-react-jsx/test/fixtures/react/adds-appropriate-newlines-when-using-spread-attribute/output.js

```diff
@@ -1,3 +1,4 @@
+/*#__PURE__*/
 React.createElement(Component, babelHelpers.extends({}, props, {
```

Comment:
Maybe `babelHelpers.extends` should get a pure annotation too?

Reply:
Maybe also other helpers are pure, I'll check 👍